### DISCUSSION TITLE
save path

### DISCUSSION THREAD

**Discussion by rss99** [mrhan1993/Fooocus-API#407] [Q&A]
As far as I can tell, when you "query" an image, it gets saved on the server running Fooocus.  Should it not be sent back to the source of the request?   How is that achieved ?

**Comment by mrhan1993** [comment]
I think you have submitted an asynchronous task. Since image generation usually takes a period of time, for the client, waiting is sometimes not necessary. The asynchronous task allows the client to leave immediately by returning the ID immediately, so that the result can be queried through the ID when needed. If it is necessary to wait for the result, a synchronous task can also be created, or polling through the ID.
